zebra-state/src/service.rs

```diff
@@ -1014,6 +1014,27 @@ impl Service<ReadRequest> for ReadStateService {
                 }
                 .boxed()
             }
+
+            // For the get_address_balance RPC.
+            ReadRequest::AddressBalance(addresses) => {
+                metrics::counter!(
+                    "state.requests",
+                    1,
+                    "service" => "read_state",
+                    "type" => "balance",
```

Comment:
Nitpick: The state also stores a chain value balance.

```suggestion
                    "type" => "address_balance",
```